docs/releases/1.9.txt

```diff
@@ -261,6 +261,27 @@ Minor features
 
 * ...
 
+:mod:`django.utils.decorators`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* :func:`~django.utils.decorators.method_decorator` can now be used to decorate
```

Comment:
I would put this in the "Generic Views" section (probably should be renamed to "Class-based views") and try to limit it to a sentence or two with a link to the section where it's explained in depth.